src/code/z_camera.c

```diff
@@ -750,41 +750,39 @@ s32 Camera_CopyPREGToModeValues(Camera* camera) {
     return true;
 }
 
-#define SHRINKWIN_MASK (0xF000)
-#define SHRINKWINVAL_MASK (0x7000)
-#define SHRINKWIN_CURVAL (0x8000)
-#define IFACE_ALPHA_MASK (0x0F00)
-
 void Camera_UpdateInterface(s16 flags) {
```

Comment:
```suggestion
void Camera_UpdateInterface(s16 interfaceFlags) {
```
for continuity with `CAM_INTERFACE_FLAGS`